plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -227,3 +228,12 @@ runPlcCekTrace values = do
         UPLC.runCek PLC.defaultCekParameters UPLC.tallying UPLC.logEmitter (p ^. UPLC.progTerm)
   res <- fromRightM (throwError . SomeException) result
   pure (logOut, tally, res)
+
+-- | A group of golden tests including UPLC size and UPLC and PIR readable.
+goldenTests :: String -> CompiledCode a -> [TestTree]
```

Comment:
Consider renaming to "goldenOpt". This is the group of test you'd want when you turn on the simplifier and want to verify that the program is optimized correctly. It makes little sense to use it on tests where the simplifier is disabled.

Reply:
I think it's also maybe just not that generic? Perhaps it should just be a local helper function in the size tests

Reply:
I originally wrote it in the size test module, but then I thought it may be useful elsewhere. I'm fine either way though.